app/support/order_row_importer.rb

```diff
@@ -223,4 +224,8 @@ def validate_product_is_importable
   def validate_user
     add_error("Invalid username or email") if user.blank?
   end
+
+  def validate_notes
+    add_error("Invalid note, 100 characters max") if note.present? && note.size > 100
```

Comment:
I'm not sure the `order.destroy` is necessary, but I think the idea makes sense.